notebooks/official/ray_on_vertex_ai/get_started_with_gemma_rov.ipynb

```diff
@@ -0,0 +1,2248 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        --machine-type={E2_HIGHCPU_32} --timeout=3600 {build_path}
</pre><p>Should it be {BUILD_MACHINE_TYPE}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
done